Koronavilkku/Koronavilkku/UI/Components/Badge.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+class Badge: UIView {
+    var horizontalInset = CGFloat(11)
+    var verticalInset = CGFloat(0)
+    var label: UILabel
+    
+    init(label: String,
+         backgroundColor: UIColor = UIColor.Primary.blue,
+         textColor: UIColor = UIColor.Greyscale.white) {
+        
+        self.label = UILabel(label: label, font: .heading4, color: textColor)
+        self.label.textAlignment = .center
+        super.init(frame: .zero)
+        
+        self.backgroundColor = backgroundColor
+        self.layer.cornerRadius = 100
+        
+        addSubview(self.label)
+
+        let insets = UIEdgeInsets(top: verticalInset,
+                                  left: horizontalInset,
+                                  bottom: verticalInset,
+                                  right: horizontalInset)
+        
+        self.label.snp.makeConstraints { [unowned self] make in
+            make.edges.equalToSuperview().inset(insets)
+            make.width.greaterThanOrEqualTo(self.label.snp.height).multipliedBy(1 / 2.4)
```

Comment:
Magic number here. Please consider using static variable and maybe add comment to clarify why this is multiplication is done